.fvm/flutter_sdk

```diff
@@ -0,0 +1 @@
+/Users/truba/fvm/versions/stable
```

Comment:
I'm not sure about using FVM for public packages.
In either case, this file should not be committed, as this is specific to `truba`-s computer.